tests/test_variables.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 NYU.
+#
+# ultraviolet is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+# Tests  to check configuration settings
+from invenio_app.factory import create_app
+
+
+# Check that configuration variables read correctly from os
+def test_var_assigned(monkeypatch):
+    """Mocking setting configuration using environment varuables"""
+    monkeypatch.setenv("APP_ALLOWED_HOSTS", "ultraviolet.dlib.nyu.edu")
+    monkeypatch.setenv(
+        "SQLALCHEMY_DATABASE_URI", "postgresql+psycopg2://test:test@somehost.com/test"
+    )
+    monkeypatch.setenv("SITE_UI_URL", "ultraviolet.dlib.nyu.edu")
+    monkeypatch.setenv("SITE_API_URL", "ultraviolet.dlib.nyu.edu/api")
+    app = create_app()
+    assert app.config.get("APP_ALLOWED_HOSTS") == "ultraviolet.dlib.nyu.edu"
+    assert (
+        app.config.get("SQLALCHEMY_DATABASE_URI")
+        == "postgresql+psycopg2://test:test@somehost.com/test"
+    )
+    assert app.config.get("SITE_UI_URL") == "ultraviolet.dlib.nyu.edu"
+    assert app.config.get("SITE_API_URL") == "ultraviolet.dlib.nyu.edu/api"
```

Comment:
In this test, it seems that multiple values are repeated (once to set them up as env. variables, another time to perform the assertion). I think it would be a good idea to save them in a variable. For instance:

```python
# Some variables assignments here
allowed_host = "ultraviolet.dlib.nyu.edu"
# Some monkey-patching here
monkeypatch.setenv("APP_ALLOWED_HOSTS", allowed_host)
# Some assertions here
assert app.config.get("APP_ALLOWED_HOSTS") == allowed_host
```

Reply:
Submitted changes